recipes-debian/squashfs-tools/squashfs-tools/fix-compat.patch

```diff
@@ -0,0 +1,63 @@
+Define FNM_EXTMATCH if not defined its glibc specific define
```

Comment:
It looks meta/recipes-devtools/squashfs-tools/squashfs-tools/fix-compat.patch and this patch, meta/recipes-devtools/squashfs-tools/squashfs-tools/squashfs-tools-4.3-sysmacros.patch and  squashfs-tools-4.3-sysmacros.patch in this pr don't have any difference. Is it okay to append FILESPATH instead of include these patches?